app/views/podcast_planner/_form_side.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="card">
+  <div class="card-header-primary">
+    <h5 class="card-title">Schedule Preview</h5>
+  </div>
+  <%= turbo_frame_tag "preview", target: "_top" do %>
+    <% if @planner.dates.present? %>
+      <%= form_with(
+            url: podcast_planner_path(@podcast),
+            method: :put
+          ) do |form| %>
+          <div class="card-body">
+            <p><%= "Publish Time: #{l(@planner.publish_time, format: :time_12_hour)}" %></p>
+            <p><%= "Number of Segments: #{@planner.segment_count}" %></p>
+            <p><%= "Number of Drafts: #{@planner.dates.count}" %></p>
+            <ul class="list-group list-group-numbered">
+              <% @planner.dates.each do |date| %>
+                <li class="list-group-item"><%= l(date, format: :day_and_date) %></li>
+              <% end %>
+            </ul>
+          </div>
+          <div class="card-footer">
+            <%= form.hidden_field :week_condition, value: @planner.week_condition %>
```

Comment:
ok, i'm not sure if it is weird to basically re-do all of these params in these `hidden_field`s, or if there was a better way to pull the params from the other form into here without causing Turbo weirdness?

Reply:
There's probably some way to reuse the turbo fields?

But Turbo seems to work best when you write simple (if redundant) HTML, without a bunch of magic.  So I think this is a good approach.